<launch>

  <include file="$(find love_maker)/launch/robustifier.launch">
    <arg name="maximum_distance" value="0.5"/>
    <arg name="minimum_detections" value="3"/>
    <arg name="raw_detection_topic" value="/torus_detections_raw"/>
    <arg name="detection_topic" value="/torus_detections"/>
    <arg name="marker_topic" value="/torus_markers"/>
  </include>

</launch>
